cmake/setup.cmake

```diff
@@ -28,10 +28,6 @@
 option(BUILD_APKS "Should we be building Android apks for our project"
   ${BUILD_APKS})
 
-if (NOT CMAKE_BUILD_TYPE)
```

Comment:
Now by default if you don't specify you get an unoptimized non-debug build iirc.
I removed this because for Visual Studio you don't have a CMAKE_BUILD_TYPE